backend/package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "helping_hands",
+	"version": "1.0.0",
+	"main": "index.js",
+	"license": "MIT",
+	"private": false,
+	"scripts": {
+		"format": "prettier . -w",
+		"dev": "ts-node index.ts",
+		"build": "tsc"
+	},
+	"dependencies": {
+		"dotenv": "^16.4.5",
+		"express": "^4.19.2",
+		"pg": "^8.12.0"
+	},
+	"devDependencies": {
+		"@types/express": "^4.17.21",
+		"@types/node": "^20.14.5",
+		"@types/pg": "^8.11.6",
+		"prettier": "^3.3.2",
```

Comment:
Nice addition.
I hope you have installed the editor extension so that it works via editor and no need to execute script.